int-ui/int-ui-decorated-window/src/main/kotlin/org/jetbrains/jewel/intui/window/styling/IntUiDecoratedWindowStyling.kt

```diff
@@ -0,0 +1,66 @@
+package org.jetbrains.jewel.intui.window.styling
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Stable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import org.jetbrains.jewel.window.styling.DecoratedWindowColors
+import org.jetbrains.jewel.window.styling.DecoratedWindowMetrics
+import org.jetbrains.jewel.window.styling.DecoratedWindowStyle
+
+@Stable data class IntUiDecoratedWindowStyle(
+    override val colors: IntUiDecoratedWindowColors,
+    override val metrics: IntUiDecoratedWindowMetrics,
+) : DecoratedWindowStyle {
+
+    companion object {
+
+        @Composable fun light(
+            intUiDecoratedWindowColors: IntUiDecoratedWindowColors = IntUiDecoratedWindowColors.light(),
+            intUiDecoratedWindowMetrics: IntUiDecoratedWindowMetrics = IntUiDecoratedWindowMetrics(1.dp),
+        ): IntUiDecoratedWindowStyle = IntUiDecoratedWindowStyle(
+            colors = intUiDecoratedWindowColors,
+            metrics = intUiDecoratedWindowMetrics,
+        )
+
+        @Composable fun dark(
+            intUiDecoratedWindowColors: IntUiDecoratedWindowColors = IntUiDecoratedWindowColors.dark(),
+            intUiDecoratedWindowMetrics: IntUiDecoratedWindowMetrics = IntUiDecoratedWindowMetrics(1.dp),
+        ): IntUiDecoratedWindowStyle = IntUiDecoratedWindowStyle(
+            colors = intUiDecoratedWindowColors,
+            metrics = intUiDecoratedWindowMetrics,
+        )
```

Comment:
For consistency with the rest of Jewel:

```suggestion
        @Composable fun light(
            colors: IntUiDecoratedWindowColors = IntUiDecoratedWindowColors.light(),
            metrics: IntUiDecoratedWindowMetrics = IntUiDecoratedWindowMetrics(),
        ) = IntUiDecoratedWindowStyle(colors, metrics)

        @Composable fun dark(
            colors: IntUiDecoratedWindowColors = IntUiDecoratedWindowColors.dark(),
            metrics: IntUiDecoratedWindowMetrics = IntUiDecoratedWindowMetrics(),
        ) = IntUiDecoratedWindowStyle(colors, metrics)
```

(the metrics default value should be in the metrics constructor)